lib/gamelogic.rb

```diff
@@ -0,0 +1,13 @@
+class GameLogic
+  def score(player1, player2, board)
+    if board.win == 1
+      player1.score += 1
+      player1.name.green
+    elsif board.win == 2
+      plyer2.score += 1
```

Comment:
Error in spelling player2 here causes game to break when player2 wins the game. Fix it.